### PR TITLE
New article about RubyConf; Skunk v0.3.0; and Skunk v0.3.1

### PR DESCRIPTION
**Pull request by etagwerker**
Hey, 

This is an article about Skunk and RubyConf 2019 😄 

Please check it out. 

Thanks!

### CODE REVIEW

File: _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+layout: post
+title: "Escaping The Tar Pit at RubyConf 2019"
+date: 2019-12-03 10:00:00
+categories: ["quality"]
```
Comment (lubc) [inline _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown]: @etagwerker Let's use `code-quality` to be consistent (re: https://www.fastruby.io/blog/tags/code-quality)

```suggestion
categories: ["code-quality"]
```

File: _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+author: etagwerker
+---
+
+This year I had the honor to speak at [RubyConf](https://rubyconf.org) in Nashville.
+It was my second time attending the conference and first time as a speaker. I
+talked about `skunk`, a library to [calculate the StinkScore](https://github.com/fastruby/skunk)
+of a module or set of modules.
+
```
Comment (lubc) [inline _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown]: @etagwerker hm, don't you want to add a link to your talk or maybe embed the video somewhere in the article?

File: _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+layout: post
+title: "Escaping The Tar Pit at RubyConf 2019"
+date: 2019-12-03 10:00:00
+categories: ["quality"]
+author: etagwerker
+---
+
+This year I had the honor to speak at [RubyConf](https://rubyconf.org) in Nashville.
+It was my second time attending the conference and first time as a speaker. I
+talked about `skunk`, a library to [calculate the StinkScore](https://github.com/fastruby/skunk)
```
Comment (bronzdoc) [inline _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown]: Replace`a library` for  `a gem`?

File: _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown (hunk 2 above)
Comment (etagwerker) [reply _posts/2019-12-03-escaping-the-tar-pit-at-rubyconf.markdown]: @lubc haha, yeah! I started the article with that in mind and then totally forgot about it... 👍

**Pull request merged**